src/views/Dashboard.vue

```diff
@@ -95,12 +95,14 @@
                   <span>Change</span>
                 </v-btn>
               </header>
-              <v-card flat>
-                <director-list-sm
-                  :showCompleteYourFilingMessage="isIncorpAppTask"
-                  :showGrayedOut="isIncorpAppFiling"
-                />
-              </v-card>
+              <div class="scrollable-container" style="max-height: 890px">
```

Comment:
Should we use px unit here?

Reply:
FWIW, I have no opinion either way here. If this was a horizontal size then maybe I'd lean towards rem so that the browser can scale everything the same, but this is a height, and technically pages are infinitely long, so 🤷

Reply:
I don't think it's critical now as we are not thinking on mobile nor other resolutions but I think it may be a problem if the user increases their font size.

Reply:
What are you thinking as an alternative? rem?

Reply:
Yes, maybe rem. This needs to be tested changing the font size and check if it fits everything.

Reply:
FWIW, most of our dialog widths are in rem.

Reply:
Totally, i'll give it a try!

Reply:
Works good!